composer.json

```diff
@@ -3,29 +3,30 @@
 	"description": "Set of functionalities enable Arabic website developers to serve professional search, present and process Arabic content in PHP",
 	"type": "library",
 	"license": "LGPL-3.0",
-	"homepage": "https://github.com/khaled-alshamaa/ar-php",
-
+	"homepage": "https://www.ar-php.org/",

```

Comment:
Not yet :-( 
That website needs to revamp first, it is safer to redirect new users to this GitHub project repository instead, at least for the time being ;-)